src/core/hle/service/nvflinger/nvflinger.cpp

```diff
@@ -232,6 +236,23 @@ const VI::Layer* NVFlinger::FindLayer(u64 display_id, u64 layer_id) const {
     return display->FindLayer(layer_id);
 }
 
+VI::Layer* Service::NVFlinger::NVFlinger::FindOrCreateLayer(u64 display_id, u64 layer_id) {
```

Comment:
Since this is already within the Service::NVFlinger namespace
```suggestion
VI::Layer* NVFlinger::FindOrCreateLayer(u64 display_id, u64 layer_id) {
```